### PR TITLE
Small lore updates

### PR DESCRIPTION
**Pull request by MciReloaded**
## About The Pull Request
Makes some medium and small level updates to lore-compliance in item descriptions, as well as adding new alert levels.

## Why It's Good For The Game
Makes lore compliance a bit better across the board, and allows for calling of emergencies for different departments. 

## Changelog
:cl:
add: Yellow Level Alert (With colors!)
add: Violet Level Alert (With colors!)
fix: Changed anything referring to IH or NanoTrasen Labs with Aegis or Lazarus. You're welcome, nerds.
fix: Removed the annoying bug with Obelisks. That's right baby, no more empty text boxes! 
fix: Gave Aegis the ability to speak German again.
imageadd: Added in overlays for the fire alarm to display yellow and violet alerts

Please. I've been in the lore lab forever. I'm losing my grip on rea-

The NEV Northern Light is a top of the line vessel refurbished by NanoTrasen for-
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/superior_animal/roach/types/nitro.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/mob/living/carbon/superior_animal/roach/nitro
```
Comment (drexample) [inline code/modules/mob/living/carbon/superior_animal/roach/types/nitro.dm:1]: why is the roach added here again?

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -540,8 +540,8 @@
 	icon_state = "frozenstar"
 
 /obj/structure/sign/faction/moebius
-	name = "NanoTrasen Laboratories"
-	desc = "A shady megacorporation that owns the Northern Light "
+	name = "Lazarus Laboratories"
+	desc = "A shady megacorporation that has a stake in the Northern Light's expedition. "
```
Comment (drexample) [inline code/game/objects/structures/signs.dm:544]: Lazarus is technically not a megacorp

File: code/game/objects/structures/signs.dm [hunk 3]
```diff
@@ -502,7 +502,7 @@
 	icon_state = "interrogation"
 
 /obj/structure/sign/department/commander
-	name = "IHS COMMANDER"
+	name = "AEGIS COMMANDER"
```
Comment (drexample) [inline code/game/objects/structures/signs.dm:505]: Aegis is not an acronym

File: code/game/objects/items/weapons/power_cells.dm [hunk 4]
```diff
@@ -217,7 +217,7 @@
 
 /obj/item/weapon/cell/small/moebius
 	name = "NanoTrasen \"Power-Geyser 120S\""
-	desc = "NanoTrasen Laboratories branded rechargeable S-standardized power cell. This one is cheap, yet better than Aster model for same price."
+	desc = "Lazarus Laboratories branded rechargeable S-standardized power cell. This one is cheap, yet better than Aster model for same price."
```
Comment (drexample) [inline code/game/objects/items/weapons/power_cells.dm:220]: Description says Lazarus and Aster (which is FTU in our lore) while the name still says NanoTrasen

**Review by drexample — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/superior_animal/roach/types/nitro.dm (hunk 1 above)
Comment (MciReloaded) [reply code/modules/mob/living/carbon/superior_animal/roach/types/nitro.dm:1]: The head of this branch was Benzin, so it got added twice. Fixed in future branches.

File: code/game/objects/structures/signs.dm (hunk 2 above)
Comment (MciReloaded) [reply code/game/objects/structures/signs.dm:544]: I did what the lore team said-

File: code/game/objects/structures/signs.dm (hunk 3 above)
Comment (MciReloaded) [reply code/game/objects/structures/signs.dm:505]: Neither is commander-

Comment (drexample) [reply code/game/objects/structures/signs.dm:505]: fair

File: code/__HELPERS/manifest.dm [hunk 5]
```diff
@@ -17,8 +17,8 @@
 
 		list("names" = list(), "header" = "Command Staff", "flag" = COMMAND),
 		list("names" = list(), "header" = "Cobalt Aegis Security", "flag" = IRONHAMMER),
-		list("names" = list(), "header" = "NanoTrasen Medical", "flag" = MEDICAL),
-		list("names" = list(), "header" = "NanoTrasen Research", "flag" = SCIENCE),
+		list("names" = list(), "header" = "Lazarus Medical", "flag" = MEDICAL),
+		list("names" = list(), "header" = "LAzarus Research", "flag" = SCIENCE),
```
Comment (drexample) [inline code/__HELPERS/manifest.dm:21]: LAzarus

Comment (MciReloaded) [reply code/__HELPERS/manifest.dm]: Fixed

**Pull request merged**